packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts

```diff
@@ -0,0 +1,39 @@
+import { SfProject } from '@salesforce/core';
+import { ConfigUtil, workspaceUtils } from '../../../src';
+
+describe('ConfigUtil', () => {
+  describe('getProjectPackageNames', () => {
+    const mockSfProject: any = {
+      getUniquePackageNames: jest
+        .fn()
+        .mockImplementation(() => ['project1', 'project2'])
+    };
+    const sfProjectGetInstanceMock = jest.fn();
+    const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
+    let getRootWorkspacePathStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      SfProject.getInstance = sfProjectGetInstanceMock;
```

Comment:
Is there a reason we couldn't do 
```
getUniquePackageNamesMock = jest.fn().mockReturnsValue(fakeProjects);
getInstanceMock = jest.spyOn(SFProject, 'getInstance').mockReturnsValues({
    getUniquePackageNames: getUniquePackageNamesMock
});
```

Reply:
nope - looks cleaner!  I had to add an "as any" to the end to get this to work, otherwise TS was beefing:

<img width="400" alt="2022-10-20_15-55-48" src="https://user-images.githubusercontent.com/46458081/197075414-c41bd84d-6bff-4fdf-92d9-1844335807e5.png">